app/README.md

```diff
@@ -2,11 +2,14 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
 
```

Comment:
Can you add a statement across the top indicating that this README has been modified from the auto-generated one?